products/cloudrun/api.yaml

```diff
@@ -260,6 +263,11 @@ objects:
       Is required when creating resources. Name is primarily intended
       for creation idempotence and configuration definition. Cannot be updated.
       More info: http://kubernetes.io/docs/user-guide/identifiers#names
+  - !ruby/object:Api::Type::String
+    name: vpc_connector
+    description: The VPC connector the cloud run instance will use
+    required: false
```

Comment:
This is fine, though not strictly necessary. Typically we just omit this if it's not required.